src/CanisterAssets/lib.mo

```diff
@@ -71,6 +122,7 @@ module {
         };
       };
       _assets.put(id, asset);
```

Comment:
why is this only called on streamAsset but not addAsset?

Reply:
added to `addAsset` too